src/gobapi/dump/to_db.py

```diff
@@ -246,6 +246,90 @@ def dump_to_db(self, full_dump=False):
         yield from self._rename_tmp_table()
         yield from self._create_indexes(model)
 
+    def create_utility_view(self):
+        """Creates view with utility columns for relating without relation table
+
+        View contains all columns from the main table, plus the RELATION_id, RELATION_VOLGNUMMER, RELATION_ref
+        and RELATION_bronwaarde columns for each RELATION (for example ligt_in_buurt_id, ligt_in_buurt_ref and
+        ligt_in_buurt_bronwaarde)
+
+        :return:
+        """
+
+        main_alias = self.model['abbreviation'].lower()
+        src_has_states = self.model.get('has_states', False)
+
+        # Collect all necessary joins and select statements
+        joins = []
+        selects = [f'{main_alias}.*']
+
+        def ref(rel_alias: str, dst_has_states: bool):
+            return f"{rel_alias}.dst_id || '_' || {rel_alias}.dst_volgnummer" \
+                if dst_has_states else f"{rel_alias}.dst_id"
+
+        for relation in [k for k in self.model['references'].keys()]:
+            # Add a join and selects for each relation
```

Comment:
why the list comprenhension and not for relation in self.model['references'].keys() ?

Reply:
or just define relations:
[get_relation_name(GOBModel(), self.catalog_name, self.collection_name, relation) for relation in self.model['references'].keys() if relation]

Reply:
Haha. That is actually a really good question.